core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -122,28 +208,44 @@ object JsonAST {
       case _ => false
     }
 
-    /** Return nth element from JSON.
-      * Meaningful only to JArray, JObject and JField. Returns JNothing for other types.
-      * <p>
-      * Example:<pre>
-      * JArray(JInt(1) :: JInt(2) :: Nil)(1) == JInt(2)
-      * </pre>
-      */
+    /**
+     * Return `n`th element from a `JArray` or `JObject` and return `JNothing` for any other kind of `JValue`.
+     *
+     * For example:
+     *
+     * {{{
+     * > val array = JArray(JInt(1) :: JInt(2) :: Nil)
+     * > array(1)
+     * JInt(1)
+     * }}}
+     */
     def apply(i: Int): JValue = JNothing
 
-    /** Return unboxed values from JSON
-      * <p>
-      * Example:<pre>
-      * JObject(JField("name", JString("joe")) :: Nil).values == Map("name" -> "joe")
-      * </pre>
-      */
+    /**
+     * Return unboxed values from JSON.
```

Comment:
May be worth expanding slightly on what “unboxed values” means here (i.e., the “contained” value in a native Scala type).